emperor/support_files/js/controller.js

```diff
@@ -224,6 +225,8 @@ define([
                                          ColorViewController);
     this.controllers.visibility = this.addTab(this.sceneViews[0].decViews,
                                               VisibilityController);
+    this.controllers.shape = this.addTab(this.sceneViews[0].decViews,
```

Comment:
I think you forgot to fix this conflict. Also, this is the container that I was referring to with all the controllers i.e. color, visibility and now shape.